client/src/index.css

```diff
@@ -9,4 +9,5 @@
 .modal {
   width: clamp(10%, 500px, 50%);
   min-height: 400px;
+  max-height: 100%;
 }
```

Comment:
general comment - anytime you see the red theta looking symbol at the bottom of a file means that it's missing a newline - best practice is to have every file end in a new line.  [An explanation on why](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3).

![image](https://user-images.githubusercontent.com/5304277/212003201-5f27eb12-608e-4f38-a040-61bacc5608ea.png)